### PR TITLE
Create 2023-05-01-console.md

### PR DESCRIPTION
**Pull request by danieltprice**
- Add 05-01 console release notes
https://neon-next-git-dprice-05-01-console-release-notes-neondatabase.vercel.app/docs/release-notes
- Update **Connection Details** screen captures and related text for the addition of **Pooled connection** and **Direct connection** tabs. For example:
https://neon-next-git-dprice-05-01-console-release-notes-neondatabase.vercel.app/docs/connect/connect-from-any-app

### CODE REVIEW

File: content/docs/connect/connect-from-any-app.md [hunk 1]
```diff
@@ -21,22 +21,22 @@ You can obtain the connection details you require from the **Connection Details*
...
-You can enable the **Pooler** option if you require a pooled connection string. The **Pooler** option adds `-pooler` to the hostname, which tells Neon to use a pooled connection. For more information, see [Connection pooling](/docs/connect/connection-pooling#connection-pooling).
+Neon supports pooled and direct connections to the database. Use a pooled connection string if your application uses a high number of concurrent connections. For more information, see [Connection pooling](/docs/connect/connection-pooling#connection-pooling).
 
 A Neon connection string includes the role, the compute endpoint hostname, and the database name.
 
 ```text
-postgres://daniel:<password>@ep-restless-rice.us-east-2.aws.neon.tech/neondb
-             ^                          ^                               ^
-             |- <role>                  |- <hostname>                   |- <database>
+postgres://daniel:<password>@ep-mute-rain-952417.us-east-2.aws.neon.tech/neondb
```
Comment (duskpoet) [inline content/docs/connect/connect-from-any-app.md:29]: should we also note the endpoint id in the conn string?

File: content/docs/connect/connect-from-any-app.md [hunk 2]
```diff
@@ -21,22 +21,22 @@ You can obtain the connection details you require from the **Connection Details*
...
+             ^                                   ^                         ^
+             |- <role>                           |- <hostname>             |- <database>
 ```
 
 You can use the details from the connection string or the connection string itself to configure a connection. For example, you might place the connection details in an `.env` file, assign the connection string to a variable, or pass the connection string on the command-line, as shown:
 
 `.env` file:
 
 ```text
-PGHOST='ep-restless-rice.us-east-2.aws.neon.tech'
```
Comment (duskpoet) [inline content/docs/connect/connect-from-any-app.md:39]: I think restless rice is an incredible foundation for a new comicbook series :D

File: content/release-notes/2023-05-01-console.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+### What's new
+
+- Control Plane: Added support for configuring the PostgreSQL [default_text_search_config](https://www.postgresql.org/docs/current/runtime-config-client.html#GUC-DEFAULT-TEXT-SEARCH-CONFIG) configuration parameter, which determines the behavior of text search functions and operators that do not have an explicit configuration. This parameter can be set for the PostgreSQL instance using the `default_endpoint_settings` property in the [Create project](https://api-docs.neon.tech/reference/createproject) or [Update project](https://api-docs.neon.tech/reference/updateproject) API, or using the `pg_settings` property in the [Create endpoint](https://api-docs.neon.tech/reference/createprojectendpoint) or [Update endpoint](https://api-docs.neon.tech/reference/updateprojectendpoint) API. Alternatively, the parameter can be set for a specific database using `ALTER DATABASE` syntax:
+
+  ```sql
+  ALTER DATABASE <dbname> SET default_text_search_config = ‘pg_catalog.english’;
+  ```
+
+- Integrations: Updated the [Neon Vercel Integration](../docs/guides/vercel) to require selecting a database and role when connecting a Vercel project to a Neon database.
```
Comment (duskpoet) [inline content/release-notes/2023-05-01-console.md:13]: this was rather a bugfix, which was not even visible to users, rather caught in tests. Let's omit it?

Comment (duskpoet) [reply content/release-notes/2023-05-01-console.md:13]: I am talking about vercel integration step

**Review by duskpoet — APPROVED** [review]

File: content/release-notes/2023-05-01-console.md (hunk 3 above)
Comment (danieltprice) [reply content/release-notes/2023-05-01-console.md:13]: Thank you. Dropped the Integrations item.

File: content/docs/connect/connect-from-any-app.md (hunk 2 above)
Comment (danieltprice) [reply content/docs/connect/connect-from-any-app.md:39]: :)

File: content/docs/connect/connect-from-any-app.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/connect/connect-from-any-app.md:29]: I added a note pointing to the definition of the connection string, which describes all the parts.
 GUI applications typically ask for a hostname, so to keep it simple, I don't call out the individual parts of the hostname here.

**Review by antonyc — APPROVED** [review]

**Pull request merged**